db/schema.rb

```diff
@@ -81,19 +81,19 @@
     t.string "headline"
   end
 
-  create_table "subscription_donations", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
-    t.uuid "subscription_id"
-    t.uuid "donation_id"
+  create_table "subscription_donations", force: :cascade do |t|
+    t.bigint "subscription_id"
+    t.bigint "donation_id"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.index ["donation_id"], name: "index_subscription_donations_on_donation_id"
     t.index ["subscription_id", "donation_id"], name: "index_subscription_donations_on_subscription_id_and_donation_id", unique: true
     t.index ["subscription_id"], name: "index_subscription_donations_on_subscription_id"
   end
 
-  create_table "subscriptions", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
-    t.uuid "user_id"
-    t.uuid "plan_id"
+  create_table "subscriptions", force: :cascade do |t|
+    t.bigint "user_id"
```

Comment:
Why we migrate from `uuid`?

Reply:
mmm this is weird, I didn't change anything on the schema. I'll drop my database and rebuild. If this comes up again then the migrations are wrong

Reply:
Looks like it was removed in this commit https://github.com/debtcollective/fundraising/commit/24c758e048a055c814e95f50af2ac081e34c0cd6#diff-1acd2e7e27a227829d5d14a91c863bb6. The schema was updated too so not sure why the update is coming up here too.

I checked the data in production and it has the same schema as this one.

![image](https://user-images.githubusercontent.com/849872/77697793-1e64ca00-6f75-11ea-8a5e-9437306b16c2.png)